.circleci/config.yml

```diff
@@ -893,12 +948,24 @@ workflows:
               only:
                 - /^test-[\d|.]+$/
                 - /^release-[\d|.]+$/
+      - code_coverage:
+          requires:
+            - prefetch-crates
```

Comment:
This is not needed for the following reasons:
- the `code_coverage` job does not restore the pre-fetched crates from cache yet
- after the restore step is added to the job above, the cache entry was created by prior jobs (during PR or auto branch during land) that touched the cache key.